docs/source/app_developers_guide/adding_a_node.rst

```diff
@@ -0,0 +1,373 @@
+
+         endpoint = "tcp://external-ip-2:8008"
+
+   #. Select static peering (the default) by setting ``peers`` to the IP
+      address and port of the first validator.
+
+      .. code-block:: none
+
+         peers = ["tcp://external-ip-1:8801"]
+
+      .. note::
+
+         If you are adding several nodes, specify the nodes in a comma-separated
+         list, as in this example:
+
+         .. code-block:: none
+
+            peers = ["tcp://external-ip-1:8801","tcp://external-ip-3:8801",...,"tcp://external-ip-5:8801"]
+
+   #. Ensure that the ``seeds`` setting is commented out.
+
+      .. note::
+
+         Static peering is appropriate for a Sawtooth network with a small
+         number of nodes (less than six or seven). Dynamic peering is preferred
+         if you plan to have many nodes on the network.
+
+         For more information on static and dynamic peering, see
+         :doc:`../sysadmin_guide/configuring_sawtooth/validator_configuration_file`
+         in the System Administrator's Guide and
+         :doc:`../cli/sawtooth-validator` in the CLI Command Reference.
+
+   #. Ensure that all other settings in this file match the settings for the
+      first validator node. These settings should be commented out unless you
+      made changes on the first node.
+
+Repeat this step to add more nodes to the Sawtooth network, if desired.
+
+Step 3: Restart the Sawtooth Network
+------------------------------------
+
+After making the configuration changes, restart all validator nodes in the
+Sawtooth network.
+
+#. Shut down the first validator node as described in
+   :ref:`stop-sawtooth-ubuntu-label`.
+
+#. Shut down the second validator node as described in
+   :ref:`stop-sawtooth-ubuntu-label`.
+
+#. Start all Sawtooth components on the first validator node. The following
+   commands are a summary of steps 3, 4, and 5 in :doc:`ubuntu`.
+
+   a. Open a terminal window and start the validator:
+
+      .. code-block:: console
+
+         user@validator$ sudo -u sawtooth sawtooth-validator -vv
+
+   #. Open a terminal window and start the REST API:
+
+      .. code-block:: console
+
+         user@rest-api$ sudo -u sawtooth sawtooth-rest-api -v
+
+   #. Start each transaction processor in a new terminal window:
+
+      .. code-block:: console
+
+         user@settings$ sudo -u sawtooth settings-tp -v
+
+      .. code-block:: console
+
+         user@intkey$ sudo -u sawtooth intkey-tp-python -v
+
+      .. code-block:: console
+
+         user@xo$ sudo -u sawtooth xo-tp-python -v
+
+      .. code-block:: console
+
+         user@validator-registry$ sudo -u sawtooth sawtooth-poet-validator-registry-tp -v
+
+#. Repeat these steps on the second validator node.
+
+
+Confirm Network Functionality
+=============================
+
+#. To check whether peering has occurred on the network, submit a block query
+   to the REST API from the first validator node.
+
+   .. code-block:: none
+
+      NEED INFO (COMMANDS AND OUTPUT)
```

Comment:
please fill-in `todo` sections

Reply:
Also, there is a `/peers` endpoint, and CLI commands to list peers, they should be listed here

Reply:
Thanks, Andrew.  Question for all reviewers (or you):  How would someone use the /peers endpoint to verify peering?  Could someone provide an example for this step? (With output, if appropriate.)  Thanks in advance.

Also should I show both "sawnet peers list" and "sawtooth peer list"? 
- "sawnet peers list" displays the peers of the specified nodes
- "sawtooth peer list" displays the addresses of a specified validator’s peers